### PR TITLE
refactor(install): move to clap

### PR DESCRIPTION
**Pull request by sylvestre**
I will do improvements once this lands

### CODE REVIEW

File: src/uu/install/src/install.rs [hunk 1]
```diff
@@ -52,14 +48,179 @@ impl Behavior {
...
+        .usage(&usage[..])
+        .arg(
+                Arg::with_name(OPT_BACKUP)
+                .long(OPT_BACKUP)
+                .help("(unimplemented) make a backup of each existing destination file")
+                .value_name("CONTROL")
+        )
+    // TODO implement flag
+        .arg(
+            Arg::with_name(OPT_BACKUP_2)
```
Comment (rivy) [inline src/uu/install/src/install.rs:99]:
```suggestion
        .arg(
            // TODO implement flag
            Arg::with_name(OPT_BACKUP_2)
```

File: src/uu/install/src/install.rs [hunk 2]
```diff
@@ -52,14 +48,179 @@ impl Behavior {
...
+                .short("Z")
+                .long(OPT_CONTEXT)
+                .help("(unimplemented) set security context of files and directories")
+                .value_name("CONTEXT")
+        )
+        .arg(Arg::with_name(ARG_FILES).multiple(true).takes_value(true))
+        .get_matches_from(args);
 
-    let matches = parse_opts(args);
+    let mut paths: Vec<String> = matches
```
Comment (rivy) [inline src/uu/install/src/install.rs:220]:
```suggestion
    let paths: Vec<String> = matches
```

File: src/uu/install/src/install.rs [hunk 3]
```diff
@@ -294,27 +322,27 @@ fn behavior(matches: &getopts::Matches) -> Result<Behavior, i32> {
...
+        "~"
     };
 
     Ok(Behavior {
         main_function,
         specified_mode,
-        suffix: backup_suffix,
-        verbose: matches.opt_present("v"),
+        suffix: backup_suffix.to_string(),
+        verbose: matches.is_present("v"),
```
Comment (rivy) [inline src/uu/install/src/install.rs:345]:
```suggestion
        verbose: matches.is_present(OPT_VERBOSE),
```

File: src/uu/install/src/install.rs [hunk 4]
```diff
@@ -52,14 +48,179 @@ impl Behavior {
     }
 }
 
+static ABOUT: &str = "Copy SOURCE to DEST or multiple SOURCE(s) to the existing
```
Comment (rivy) [inline src/uu/install/src/install.rs:51]: My local `install` manual  says...

```text
In the first three forms, copy SOURCE to DEST or multiple SOURCE(s) to the existing
DIRECTORY, while setting permission modes and owner/group. In the 4th form, create
all components of the given DIRECTORY(ies).
```

I'm not sure how best to represent that here, but I'm good with this as a starting point.
```

**Review by rivy — CHANGES_REQUESTED** [review]
Minor nits.

File: src/uu/install/src/install.rs (hunk 4 above)
Comment (sylvestre) [reply src/uu/install/src/install.rs:51]: most of the features are missing in this program, so, still a lot of work+tests :)

**Comment by sylvestre** [comment]
As it was only nit and the CI is green, I will land it now (I am building stuff on top of this)

**Comment by rivy** [comment]
> As it was only nit and the CI is green, I will land it now (I am building stuff on top of this)

I'm fine with this for the `clap` changes you're making... After collaborating on several of these, you're already incorporating most of my "nits" into your new PRs already. 😄 

I'll review more of the PRs, but it's going to continue taking a bit longer because of the COVID resurgence keeping me busier at hospital work.

**Comment by sylvestre** [comment]
> COVID resurgence keeping me busier at hospital work.

Really sorry about that :(
This is why I haven't been bothering you (but only auto-landing trivial changes :)

**Pull request merged**